h2o-py/tests/testdir_munging/pyunit_pubdev_4870_sort_bug.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+def sort():
+    df = h2o.import_file(pyunit_utils.locate("smalldata/synthetic/smallIntFloats.csv.zip"))
+    sorted_column_indices = [0,1]
+    df1 = df.sort(sorted_column_indices).asnumeric()
+    pyunit_utils.check_sorted_2_columns(df1, sorted_column_indices, prob=0.2)
+    print("wow")
+    h2o.remove_all()
```

Comment:
It is not necessary to call h2o.remove_all() after the test. This is handled by the test framework.

Reply:
Sure, will remove it.  Add it there for my computer to not blow up on me.